src/components/header/renderer.tsx

```diff
@@ -508,11 +523,8 @@ class Header extends React.PureComponent<Props, State> {
         </section>
 
         <section className="right-section">
-          <PortalWithState closeOnEsc>
+          <PortalWithState closeOnEsc ref="helpModal">
```

Comment:
String refs are deprecated in react. Can you find a different way? Do you see the deprecation notice in vscode?